docker/datahub-ingestion/Dockerfile

```diff
@@ -22,10 +22,22 @@ ENV PATH="/datahub-ingestion/.local/bin:$PATH"
 FROM base as slim-install
 RUN pip install --no-cache --user ".[base,datahub-rest,datahub-kafka,snowflake,bigquery,redshift,mysql,postgres,hive,clickhouse,glue,dbt,looker,lookml,tableau,powerbi,superset,datahub-business-glossary]"
 
-FROM base as full-install
+FROM base as full-install-build
+
+USER 0
+RUN apt-get update && apt-get install -y -qq maven
+
+USER datahub
+COPY ./docker/datahub-ingestion/pyspark_jars.sh .
+
 RUN pip install --no-cache --user ".[base]" && \
     pip install --no-cache --user "./airflow-plugin[acryl-datahub-airflow-plugin]" && \
-    pip install --no-cache --user ".[all]"
+    pip install --no-cache --user ".[all]" && \
+    ./pyspark_jars.sh
```

Comment:
if you have numbers for this - what does this do to install sizes?

Reply:
No expected (or observed impact). Still the full image is ~1.08 GB